52.md

```diff
@@ -0,0 +1,142 @@
+NIP-52
+======
+
+Calendar Events
+---------------
+
+`draft` `optional` `author:tyiu`
+
+This specification defines calendar events representing an occurrence at a specific moment or between moments. These calendar events are replaceable and referenceable per [NIP-33](33.md) and deletable per [NIP-09](09.md).
+
+Unlike the term `calendar event` specific to this NIP, the term `event` is used broadly in all the NIPs to describe any Nostr event. The distinction is being made here to discern between the two terms.
+
+## Calendar Events
+
+### Format
+
+The format uses a parameterized replaceable event kind `31923` with a list of tags as described below:
+* `d` (required) universally unique identifier (UUID). Generated by the client creating the calendar event.
+* `summary` (required) name of the calendar event
+* `description` (optional) detailed description of the calendar event
```

Comment:
should `description` not be in `content`?

Reply:
Definitely. NIP-23 already defines how `content` in long-form notes is handled (ie. markdown) so clients would already have that worked out.

Reply:
Sure. I was following the naming from the iCal spec, but `content` seems more in line with idiomatic Nostr.